src/components/radio/radio.ts

```diff
@@ -71,9 +77,13 @@ export class MdRadioGroup implements AfterContentInit, ControlValueAccessor {
   /** The currently selected radio button. Should match value. */
   private _selected: MdRadioButton = null;
 
+  /** Whether the `value` has been set to its initial value. */
+  private _isInitialized: boolean = false;
+
   /** Change event subscription set up by registerOnChange (ControlValueAccessor). */
   private _changeSubscription: {unsubscribe: () => any} = null;
 
+  /** onTouch function registered via registerOnTouch (ControlValueAccessor). */
   onTouched: () => any = () => {};
```

Comment:
This is very confusing to read.  I don't have a suggestion really, but that's entirely too many `=` in a row.